PAS-536 | Add support for `AuthenticatorDisplayName`

Tyrrrz opened the pull request:
Waiting on: https://github.com/bitwarden/passwordless-dotnet/pull/152

abergs reviewed (commented):
We need to add migrations, corect?

Tyrrrz commented:
@abergs yes, correct

Tyrrrz commented:
This needs a new release of Passwordless.NET and a package update to resolve missing references

Tyrrrz commented:
Added migrations and updated the Passwordless.NET SDK version to `2.1.0-beta.3` which is pending approval here: https://github.com/bitwarden/passwordless-devops/actions/runs/10775849985

Once that package is released, this PR should technically be good to merge. Would still need to run some smoke tests for good measure, however.

Tyrrrz commented on src/Service/Fido2Service.cs:
```diff
@@ -16,53 +16,32 @@
 namespace Passwordless.Service;
 
-public class Fido2Service : IFido2Service
+public class Fido2Service(
```
Only formatting changes here

Tyrrrz commented:
Should be ready
